docs/virtual_machines/disks_and_volumes.md

```diff
@@ -1451,14 +1451,17 @@ allocate 4 queues (to match the number of CPUs requested).
 
 ### Disk device cache
 
-KubeVirt supports `none` and `writethrough` KVM/QEMU cache modes.
+KubeVirt supports `none`, `writeback`, and `writethrough` KVM/QEMU cache modes.
 
 -   `none` I/O from the guest is not cached on the host. Use this option
     for guests with large I/O requirements. This option is generally the
     best choice.
 
+-   `writeback` I/O from the guest is cached on the host and written through
+    to the physical media when the guest OS issues a flush.
+
 -   `writethrough` I/O from the guest is cached on the host but written
-    through to the physical medium.
+    through to the physical medium on each write.
```

Comment:
I'm not 100% sure how to phrase it best. Perhaps we should say that the data is stored in the page cache on the host and writes are reported as completed to the guest only when the data has been committed to the storage device.

@alicefr what do you think?

Reply:
@vladikr yes, it sounds a more correct formulation to me. Here, we are missing that the writes are completed when they are done on the storage device.

Reply:
I updated the wording, is this more correct?

Reply:
Yes, sounds good to me